# Fix "*** No rule to make target '../../PHonon/PH/libph.a', needed by 'kcw.x'.  Stop." spurious failures
--- Makefile
+++ Makefile
@@ -138,7 +138,7 @@
 	if test -d test-suite ; then \
 	( cd test-suite ; make run-travis || exit 1 ) ; fi
 
-kcw : pwlibs lrmods pp w90lib
+kcw : pwlibs lrmods pp w90lib phlibs
 	if test -d KCW ; then \
 	( cd KCW ; $(MAKE) all || exit 1 ) ; fi
